src/loaders/zarr_utils/types.ts

```diff
@@ -93,3 +93,20 @@ export type OMEZarrMetadata = {
 };
 
 export type NumericZarrArray = zarr.Array<zarr.NumberDataType, WrappedStore<RequestInit>>;
+
+/** A record with everything we need to access and use a single remote source of multiscale OME-Zarr data. */
+export type ZarrSource = {
+  /** Representations of each scale level in this zarr. We pick one and pass it to `zarrGet` to load data. */
```

Comment:
where is zarrGet defined? Is that a zarrita native function?

Reply:
`zarrGet` is the alias I import zarrita's `get` function under in `OMEZarrLoader` - I believe they prefer if you import it as `zarr.get`, but I think I was worried about tree shaking?

Reply:
Was only wondering because of the comment it's hard to tell whether that's our code or third party code